scripts/exregional_nonvarcldanl.sh

```diff
@@ -165,6 +165,12 @@ else
     bkpath=${cycle_dir}${slash_ensmem_subdir}/fcst_fv3lam${cycle_tag}/INPUT
 fi
 
+if [ ${i_uncertainty} == ".true." ]; then
+  # Copy analysis fields into uncertainties - data will be overwritten
+  echo "EXREGIONAL_NONVARCLDANL.SH: copy tracer file into uncertainty file "
+  cp_vrfy ${bkpath}/fv_tracer.res.tile1.nc  ${bkpath}/fv_tracer.unc.tile1.nc
```

Comment:
This uncertainty only applies to cloud analysis results?

Reply:
Yes, EMC is working on uncertainties for some other basic fields, and we we work on extending the uncertainty to other aviation fields like cloud tops, ceilings, etc.